.gitlab/tests/contrib.yml

```diff
@@ -274,6 +274,40 @@ opentracer:
   variables:
     SUITE_NAME: "opentracer"
 
+aiohttp:
```

Comment:
nit: Should we have these ordered alphabetically?

Reply:
good idea!